### PR TITLE
Fix VPR_ARCH_ARGS

### PR DESCRIPTION
**Pull request by elmsfu**
VPR_ARCH_ARGS was being ignored in cmake define_arch()

ice40 tests should complete now.

Also added ability to specify conda package versions and builds. This was helpful in bisecting the failure and will be good for any release or milestone.

### CODE REVIEW

File: make/env.cmake [hunk 1]
```diff
@@ -156,7 +156,8 @@ function(ADD_CONDA_PACKAGE)
       DEPENDS conda ${CONDA_BIN}
       )
 
-    set(TARGET conda_${ADD_CONDA_PACKAGE_NAME})
+    # Allow specification of version and build, but strip it off for the package name
+    string(REGEX MATCH "^[^=]+" TARGET ${ADD_CONDA_PACKAGE_NAME})
```
Comment (litghost) [inline make/env.cmake]: This change seems unrelated?

Comment (elmsfu) [reply make/env.cmake]: It was an improvement during debugging. Do you want me to break out the one line change to a different PR?

Comment (litghost) [reply make/env.cmake]: Ya
